Office 365/o365/_meta/smart-descriptions.json

```diff
@@ -73,6 +133,31 @@
       }
     ]
   },
+  {
+    "value": "Investigation {office365.investigation.id} terminated by user {user.id}",
+    "conditions": [
+      {
+        "field": "office365.investigation.id"
+      },
+      {
+        "field": "office365.investigation.name"
+      },
+      {
+        "field": "office365.investigation.type"
+      },
```

Comment:
Is useful to test the existence of these fields?
```suggestion
```

Reply:
I'm not sure but I think so, yes, it was like that before.